FizzBuzz-2/fizzbuzz2.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+
+namespace FizzBuzz2
+{
+    public class Program2
+    {
+        public static void Main(string[] args)
+        {
+
+            for (int counter = 1; counter <= 100; counter++)
+
+            {
+                if (counter % 3 == 0 && counter % 5 != 0)
```

Comment:
Another way to accomplish this is to check for divisibility by both 5 and 3 first , then check for divisibility by the 3 or 5.

Reply:
I see. Good comment.